src/parser/csql_grammar.y

```diff
@@ -3111,6 +3116,40 @@ create_stmt
 			$$ = node;
 			PARSER_SAVE_ERR_CONTEXT ($$, @$.buffer_pos)
 
+		DBG_PRINT}}
+	| CREATE			/* 1 */
+	  opt_or_replace		/* 2 */
+	  	{ push_msg (MSGCAT_SYNTAX_SYNONYM_INVALID_CREATE); }	/* 3 */
+	  opt_access_modifier		/* 4 */
+	  SYNONYM			/* 5 */
+	  synonym_name_without_dot	/* 6 */
+	  For				/* 7 */
+	  class_name			/* 8 */
+	  opt_comment_spec		/* 9 */
+	  	{ pop_msg(); }
+		{{
+
```

Comment:
Please add below
`DBG_TRACE_GRAMMAR(create_stmt, | CREATE  opt_or_replace opt_access_modifier SYNONYM synonym_name_without_dot For class_name opt_comment_spec);`

Reply:
Thank you. Added.